debian/control

```diff
@@ -11,6 +11,7 @@ Architecture: all
 Section: python
 Depends: ${python:Depends},
          ${misc:Depends},
+         python3,
```

Comment:
Why, this application is still in python2

Reply:
there is a script that is in python3 (reported by lintian)...

Reply:
Found it: https://github.com/wazo-platform/wazo-agid/blob/master/bin/change-pass-vm

So we also missed some deps:
```
wazo-auth-client-python3
xivo-lib-python-python3
```

Reply:
added